go/cli/mcap/utils/utils.go

```diff
@@ -42,66 +57,42 @@ func StdoutRedirected() bool {
 	return true
 }
 
-func GetReader(ctx context.Context, filename string) (func() error, io.ReadSeekCloser, error) {
-	var rs io.ReadSeekCloser
-	var err error
-	closeReader := func() error { return nil }
-	scheme, bucket, path := GetScheme(filename)
-	if scheme != "" {
-		switch scheme {
-		case "gs":
-			client, err := storage.NewClient(ctx)
-			if err != nil {
-				return closeReader, nil, fmt.Errorf("failed to create GCS client: %w", err)
-			}
-			closeReader = client.Close
-			object := client.Bucket(bucket).Object(path)
-			rs, err = NewGCSReadSeekCloser(ctx, object)
-			if err != nil {
-				return closeReader, nil, fmt.Errorf("failed to build read seek closer: %w", err)
-			}
-		default:
-			return closeReader, nil, fmt.Errorf("unsupported remote file scheme: %s", scheme)
+func GetReader(ctx context.Context, uri string) (io.ReadSeekCloser, bool, error) {
```

Comment:
it would be useful to add a docstring here describing the behavior of the new bool returned